test/sleepy_runner.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_queue'
+require 'test_queue/runner/minitest'
+
+class SleepyTestRunner < TestQueue::Runner::MiniTest
+  def around_filter(*args)
+    if ENV['SLEEP_AS_RELAY'] && relay? 
+      sleep 5
+    elsif ENV['SLEEP_AS_MASTER'] && !relay?
+      sleep 5
+    end
```

Comment:
I don't think this will guarantee that a particular master's workers will run the failing suite. `around_filter` gets called inside each worker just after receiving the suite to run, so it seems like workers will be handed an initial suite to run and *then* will sleep. I.e., imagine the following execution when we're trying to get the remote master to run the failing suite:

1. `SLEEP_AS_MASTER=1`
2. Central master boots and spawns workers
3. Central master hands the failing suite to one of its own workers
4. Central master's workers sleep
5. Remote master boots and connects to central master
6. Central master hands all the rest of the suites to the remote master's workers
7. Remote master's workers run those suites and remote master exits with success
8. Central master's workers wake up and run their suites, including the failing one

For the sleeping to be effective I think it needs to happen before suites are assigned, i.e., before the workers send their first `POP` messages.